DQMServices/Components/scripts/dqm-access

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
was this ever test after migration, see https://cms-talk.web.cern.ch/t/dqm-access-script-is-broken-in-recent-cmssw-releases/22637